### PR TITLE
[irods/irods#7312] Get package revision from IrodsCPackCommon

### PR DESCRIPTION
**Pull request by SwooshyCueb**

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Add the pound.

**Review by alanking — DISMISSED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -269,16 +269,12 @@ set(CPACK_DEB_COMPONENT_INSTALL OFF)
 set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS OFF)
 set(CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION ON)
 set(CPACK_DEBIAN_PACKAGE_RELEASE "${IRODS_PACKAGE_REVISION}")
```
Comment (alanking) [inline CMakeLists.txt:271]: Is this supposed to be removed? Just noticed that it is being removed in the other PRs.

Comment (SwooshyCueb) [reply CMakeLists.txt]: Nice catch

**Review by korydraughn — APPROVED** [review]

**Comment by korydraughn** [comment]
Let's wait for one more approval.

**Review by alanking — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**